src/app/modules/map/component/map-dialog/map-dialog.component.html

```diff
@@ -20,60 +20,53 @@
       <div>{{ 'map.not-found' | translate }}</div>
     </ng-container>
     <ng-container *ngIf="map">
-      <div
-        class="clickable"
-        (click)="onMapClick($event)"
-        [title]="
-          (data.settings.evaluateBrowserAlwaysExternal
-            ? 'app.clickable-external-only'
-            : 'app.clickable'
-          ) | translate
-        "
-      >
+      <div class="description">
         <ng-container *ngIf="map.layout">
-          <div class="description">
-            {{ map.layout }}
-            <app-item-frame-separator
-              [item]="data.item"
-              *ngIf="
-                map.bossCount > 0 ||
-                (map.bosses && map.bosses.length > 0) ||
-                map.encounter ||
-                (map.items && map.items.length > 0)
-              "
-            >
-            </app-item-frame-separator>
-          </div>
+          <mat-expansion-panel>
+            <mat-expansion-panel-header expandedHeight="20px" collapsedHeight="20px">
+              <mat-panel-title>{{ 'map.map-layout' | translate }}</mat-panel-title>
+            </mat-expansion-panel-header>
+            <span [innerHtml]="nl2br(map.layout)"></span>
+          </mat-expansion-panel>
+          <app-item-frame-separator [item]="data.item"
+                                    *ngIf="map.bossCount > 0 || (map.bosses && map.bosses.length > 0) || map.encounter || (map.items && map.items.length > 0) || map.url">
```

Comment:
there is no reason to `> 0` 0 is falsey on its own.

Reply:
you can also `map.bosses?.length` instead of null checking manually

Reply:
Fixed/changed all of these in the component.